### PR TITLE
Issue#96

### PR DESCRIPTION
**Pull request by tareq89**
- job track page made public
- order info, order status, tracking info, package info panels on job track page
- tracking info contains map and assigned assets name, photo, phone number
- job tasks are now typed
- order status is now created by running a loop on job tasks
- job tasks time details, like pickup started, end time, delivery started, end time added but commented,
should be uncommented once this job task start, completed time is supported on TaskCat
- job track page panels are now collapsible

### CODE REVIEW

File: app/job/job-track/job-track.component.html [hunk 1]
```diff
@@ -253,6 +228,10 @@ <h3 class="panel-title">
 					</div>
 				</div>
 			</div>
+			<div>
+				<p class="text-center">For any issue or query contact <b>GO! Fetch</b> at <b><a href="tel:">01735829687</a></b></p>
```
Comment (Shaphil) [inline app/job/job-track/job-track.component.html]: Should be
    `<a href="tel:01735829687">01735829687</a>`
I suppose

File: app/job/shared/jobtasks.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        this.ModifiedTime = new Date(task["ModifiedTime"]);
+        this.CompletionTime = new Date(task["CompletionTime"]);
+        this.IsReadytoMoveToNextTask = task["IsReadytoMoveToNextTask"];
+        this.IsDependencySatisfied = task["IsDependencySatisfied"];
+        this.IsStartingTask = task["IsStartingTask"];
+        this.IsTerminatingTask = task["IsTerminatingTask"];
+        this.ETAFailed = task["ETAFailed"];
+    }
+}
+
```
Comment (Shaphil) [inline app/job/shared/jobtasks.ts]: Are _From_ and _To_ fields not necessary for every **JobTask**?

File: app/job/shared/orderInfo.service.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export class OrderInfoService {
         switch (statusNumber) {
             case 1:
                 _orderInfoHeading = "Your Order is created, we will start processing it shortly!";
-                _orderInfoDesc = "We've got your order, we are getting it ready, a confirmation of your order will be sent to <b>"+ job.User.Email +".</b>";
+                _orderInfoDesc = "We've got your order, we are getting it ready";
```
Comment (Shaphil) [inline app/job/shared/orderInfo.service.ts]: hard coded text? If there are a lots of text information to show to the user in many different places, may be a separate JSON file is suitable? Just a suggestion, I don't know the Angular way.

File: app/job/shared/jobtasks.ts (hunk 2 above)
Comment (thehoneymad) [inline app/job/shared/jobtasks.ts]: Um...no, it shouldn't be according to TaskCat.

**Pull request merged**